crates/ruff/resources/test/fixtures/pycodestyle/E731.py

```diff
@@ -1,51 +1,135 @@
-#: E731
-f = lambda x: 2 * x
-#: E731
-f = lambda x: 2 * x
-#: E731
-while False:
-    this = lambda y, z: 2 * x
-#: E731
-f = lambda: (yield 1)
-#: E731
-f = lambda: (yield from g())
-#: E731
-class F:
+def scope():
+    # E731
     f = lambda x: 2 * x
```

Comment:
Every case here is now in its own scope, to avoid interference with the shadowing rules we're enforcing.